CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Unreleased
```

Comment:
appreciate you doing this- i've considered asking for changelog entries in PRs but... it turns out that the merge conflicts get really gnarly. gonna remove this and then merge this in